Add replica info to maintenance (#1592)
timescale/docs: cloud/service-operations/maintenance.md

applied when your service is resumed.
</highlight>

## Replicas and maintenance

Services with replicas do not require maintenance downtime. Instead, they have
one or two failover events during maintenance, taking less than a few seconds
each.

During a maintenance event, services with replicas perform maintenance on each
node independently. Maintenance begins on one node, and when it is finished,
that node is promoted to primary. The other node then begins maintenance, and

> when it is complete it remains the replica. Sometimes, maintenance begins with

Reviewer: ```suggestion
when it is complete, it remains the replica. Sometimes, maintenance begins with
```

> the primary, causing the replica node to be promoted at the start, meaning the
> service experiences two total promotions, or failovers, during a maintenance
> event.

Reviewer: ```suggestion
the primary. This causes the replica node to be promoted at the start. If this happens, the
service experiences two promotions, or failovers, during the maintenance
event.
```

It may be easier to parse with the shorter sentences, but not blocking
